lti_emailer/settings/local.py

```diff
@@ -15,3 +15,14 @@
 }
 
 dictConfig(LOGGING)
+
+CANVAS_URL = 'http://canvas.icommons.harvard.edu/'
```

Comment:
In general,  we should not be checking in these values into GitHub (URLs).  I would expect them to be in the secure.py file along with the user name, password and selenium grid url.